runtime/Cpp/runtime/src/RuleContext.h

```diff
@@ -110,15 +110,15 @@ namespace antlr4 {
     ///  (root child1 .. childN). Print just a node if this is a leaf.
     ///  We have to know the recognizer so we can get rule names.
     /// </summary>
-    virtual std::string toStringTree(Parser *recog) override;
+    virtual std::string toStringTree(Parser *recog, bool pretty=false) override;
 
     /// <summary>
     /// Print out a whole tree, not just a node, in LISP format
     ///  (root child1 .. childN). Print just a node if this is a leaf.
     /// </summary>
-    virtual std::string toStringTree(std::vector<std::string> &ruleNames);
+    virtual std::string toStringTree(std::vector<std::string> &ruleNames, bool pretty=false);
```

Comment:
Please use a space on each side of the equal sign (do this for all occurences in this patch).

Reply:
fixed